### ISSUE TITLE
Linux: Cannot find module (gamepad.node Binary)

### ISSUE THREAD

**Issue by ExploitSage** [creationix/node-gamepad#4]
I am working on a Control System for Robotics using Node-Webkit (inheriting the project is a more accurate description). I honestly barely know how node (or nw) work and am basically stumbling through this learning as I go. 

ran
`npm install gamepad`

placed
`var gamepad = require("gamepad");`
in one of the js files, there all the other dependencies are.
I started up nw, and got a nice little error:

```
Uncaught node.js Error 

Error: Cannot find module '/home/gixxy/Desktop/DriverStation.js/node_modules/gamepad/build/gamepad/v1.0.2/Release/node-webkit-v0.10.5-linux-x64/gamepad.node'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:273:25)
    at Module.require (module.js:362:17)
    at require (module.js:378:17)
    at Object.<anonymous> (/home/gixxy/Desktop/DriverStation.js/node_modules/gamepad/gamepad.js:4:15)
    at Module._compile (module.js:454:26)
    at Object.Module._extensions..js (module.js:472:10)
    at Module.load (module.js:354:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:362:17)
```

Basically the last part of the path is wrong, on my system it is at:
`node-v11-linux-x64`
not
`node-webkit-v0.10.5-linux-x64`

I am using Arch Linux, here is my uname -a:
`Linux ASUS14-ARCH 3.17.3-1-ARCH #1 SMP PREEMPT Fri Nov 14 23:13:48 CET 2014 x86_64 GNU/Linux`

Any and all help is appreciated. If you want to know what kind of robots this is going to be used for: [YouTube](https://www.youtube.com/watch?v=bkooHbifvfk)
